applications/crossbar/src/modules/cb_clicktocall.erl

```diff
@@ -424,21 +424,23 @@ build_originate_req(Contact, Context) ->
                                                    ,route = Exten}),
 
     lager:debug("attempting clicktocall ~s in account ~s", [FriendlyName, AccountId]),
+    {'ok', AccountDoc} = couch_mgr:open_cache_doc(?WH_ACCOUNTS_DB, AccountId),
```

Comment:
This should have been `kz_account:fetch(AccountId)` as the account's version is considered authoritative and not the `accounts` version.